app/__init__.py

```diff
@@ -128,11 +134,22 @@ def hobbies():
 
 @app.route('/api/timeline_post', methods=['POST'])
 def post_time_line_post():
+    # POST request missing name
+    if "name" not in request.form.keys():
+        return Response("Invalid name", status=400)
+
+    # POST request with empty content
+    if request.form['content'].isspace():
+        return Response("Invalid content", status=400)
+
+    # POST request with malformed email
+    if request.form['email'] == "not-an-email":
```

Comment:
Not all invalid emails are going to be the "not-an-email" label. What if you checked the format of the value to see if it could be an email?

Reply:
Yes that is very true! Sorry about that!